lib/cocoapods/command/setup.rb

```diff
@@ -3,100 +3,15 @@
 module Pod
   class Command
     class Setup < Command
-      self.summary = 'Setup the CocoaPods environment'
+      self.summary = 'Legacy command present for compatibility reasons - deprecated in 1.8.0'
```

Comment:
is this needed? maybe the command can be empty and do nothing instead. What if we do use this command for anything in the future?

Reply:
I am thinking that people sometimes use the command in CI and we don't want their flows to break. On the other hand, future use seems a bit YAGNI. 

What would you like me to do exactly?

Reply:
Not sure. I figured a `pod setup` command could prove useful to us in the future.

I completely agree we dont want to break flows and this doesn't seem to be a breaking changes. I am not arguing to delete it, I was more trying to avoid us saying "its deprecated" and in the next version "oh no wait its not deprecated afterall".

Can let others weigh in on this can be fine as is.

Reply:
Makes sense - I'll make it totally silent.